csharp/src/Google.Protobuf.Test/GeneratedMessageTest.Proto2.cs

```diff
@@ -261,6 +261,13 @@ public void RequiredFields()
             Assert.True(message.IsInitialized());
         }
 
+        [Test]
+        public void RequiredFieldsNoThrow()
+        {
+            TestRequired.Parser.ParseFrom(new byte[0]);

```

Comment:
without context, it's not obvious what this test does and what is being checked. 
can we add comment or improve clarity?
Also consider using `Assert.DoesNotThrow` to make it obvious what is it that you're testing.